### PR TITLE
Remove set_sensor_enabled from HolodeckEnvironment and documentation

### PR DESCRIPTION
**Pull request by vinhowe**
One half of fixing #268&mdash;the engine PR is https://github.com/BYU-PCCL/holodeck-engine/pull/208.

I'll add a note in the changelog once we verify that these are the changes we want to make.

### CODE REVIEW

File: src/holodeck/environments.py [hunk 1]
```diff
@@ -601,20 +601,6 @@ def set_control_scheme(self, agent_name, control_scheme):
...
-
-        Args:
-            agent_name (:obj:`str`): The name of the agent whose sensor will be switched
-            sensor_name (:obj:`str`): The name of the sensor to be switched
-            enabled (:obj:`bool`): Boolean representing whether to enable or disable the sensor
-        """
-        if agent_name not in self._sensor_map:
-            print("No such agent %s" % agent_name)
-        else:
-            command_to_send = SetSensorEnabledCommand(agent_name, sensor_name, enabled)
```
Comment (jaydenmilne) [inline src/holodeck/environments.py]: Do you need to delete the `SetSensorEnabledCommand` class?

Comment (vinhowe) [reply src/holodeck/environments.py]: Yes--good catch.

File: docs/changelog/changelog.rst [hunk 2]
```diff
@@ -54,17 +54,21 @@ Changes
   the environment to exit rather than fail silently.
   This includes all relevant methods in the
   :class:`~holodeck.weather.WeatherController`.
+- Removed the ability to toggle sensors during runtime with the removal of
+  :class:`~holodeck.environments.command.SetSensorEnabledCommand`,
+  :meth:`~holodeck.environments.HolodeckEnvironment.set_sensor_enabled`, and
+  :meth:`~holodeck.sensors.HolodeckSensor.set_sensor_enable`.
```
Comment (jaydenmilne) [inline docs/changelog/changelog.rst]: Add a note saying they can make a custom configuration without a sensor if they don't want to use it

Comment (vinhowe) [reply docs/changelog/changelog.rst]: @jaydenmilne does this work?

**Pull request merged**